Geometry/MuonNumbering/src/ME0NumberingScheme.cc

```diff
@@ -73,9 +76,11 @@ int ME0NumberingScheme::baseNumberToUnitNumber(const MuonBaseNumber& num) {
 
 
 
-  int maxLevel = theLayerLevel;
+  int maxLevel = 0;
+  if(theNEtaPart==1)  maxLevel = theLayerLevel;
+  if(theNEtaPart==10) maxLevel = theRollLevel;
```

Comment:
maybe "else" instead (or "else if (theNEtaPart>1)") ?
so that next time the layout changes, there is no need to edit this file